src/main/java/pl/touk/sputnik/processor/spotbugs/SpotBugsProcessor.java

```diff
@@ -121,4 +130,23 @@ private String getExcludeFilterFilename() {
         log.info("Using SpotBugs exclude filter file {}", excludeFilterFilename);
         return excludeFilterFilename;
     }
+
+    private void loadAllSpotbugsPlugins() {
+        String pluginsLocation = config.getProperty(GeneralOption.SPOTBUGS_PLUGINS_LOCATION);
+        if (pluginsLocation != null) {
+            File[] pluginsList = new File(pluginsLocation).listFiles();
```

Comment:
I think this is still a file, not yet a loaded plugin. Maybe `fileList` and `File file` as iterator below would be less confusing?